src/components/ImageCard.react.js

```diff
@@ -147,8 +159,52 @@ var ImageCard = React.createClass({
         <span className="icon icon-badge-private"></span>
       );
     }
-    return (
-      <div className="image-item">
+    let create;
+    let overlay;
+    if (this.props.image.is_local) {
```

Comment:
With the look and feel of local vs non-local images being different, I'm not sure how to get around the big if/else statement. 
React wont allow partials that aren't properly nested. Any suggestions?